google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/SelectConfigDestinationFolderDialog.java

```diff
@@ -63,19 +64,14 @@ public SelectConfigDestinationFolderDialog(
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
 
-    // Present a canonical target folder as default in the path field.
-    if (project != null && project.getBasePath() != null) {
-      if (fileType == ConfigFileType.APP_YAML) {
-        Path appYamlPath = Paths.get(project.getBasePath()
-            + AppEngineDeploymentRunConfigurationEditor.DEFAULT_APP_YAML_DIR);
-        destinationFolderChooser.setText(appYamlPath.toString());
-      } else if (fileType == ConfigFileType.DOCKERFILE) {
-        Path dockerfilePath = Paths.get(project.getBasePath()
-            + AppEngineDeploymentRunConfigurationEditor.DEFAULT_DOCKERFILE_DIR);
-        destinationFolderChooser.setText(dockerfilePath.toString());
-      } else {
-        destinationFolderChooser.setText(project.getBasePath());
+    // Suggests the parent of the specified file as directory.
```

Comment:
Why does this no longer suggest the canonical paths for these files? What if filePath is empty (e.g. the user has no Dockerfile/app.yaml entered)?